.github/workflows/verify.yml

```diff
@@ -943,7 +945,7 @@ jobs:
   #
   docgen:
     needs: [source-archive-with-subprojects, source-format-check, build-python]
-    runs-on: [self-hosted, linux, verify]
```

Comment:
Why is the one on line 644 `[self-hosted, linux, x64, verify]`, but this one "only" `[self-hosted, linux, verify]`?

Reply:
I believe the 'runs-on' label for the verify-job was updated when the arch-runner was added to avoid scheduling it there. The docgen job should have also been set to x86 at that time.

Regardless, it's sufficient to schedule it on 'runs: ubuntu-latest' now, as all GitHub-hosted runners are x86, except for macOS, which these jobs won't use.